CMakeLists.txt

```diff
@@ -130,6 +131,10 @@ set(FACTER_CXX_FLAGS "${LEATHERMAN_CXX_FLAGS}")
 
 add_definitions(${LEATHERMAN_DEFINITIONS})
 
+if (STATIC_FACTER)
```

Comment:
This is usually set by how Leatherman is built, so doesn't make a lot of sense to include here.